domains/bhavitha.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Portfolio website",
+    "repo": "https://github.com/nightcoder26/bhavitha.github.io",
+    "owner": {
+        "username": "nightcoder26",
+        "email": "",
+        "twitter": ""
+    },
+    "record": {
+        "CNAME": "nightcoder.github.io"
```

Comment:
I can not reach your website, 404 not found and i don't see any repository on the user that owns this github pages url